pkg/ui/workspaces/crdb-js-proto/BUILD.bazel

```diff
@@ -0,0 +1,81 @@
+load("@aspect_rules_js//js:defs.bzl", "js_binary", "js_run_binary", "js_test")
+load("@aspect_rules_js//npm:defs.bzl", "npm_package")
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("//docs/generated/http:defs.bzl", "PROTOBUF_TARGETS")
+load("//pkg/ui/workspaces/db-console/src/js:defs.bzl", "proto_sources")
+
+npm_link_all_packages(name = "node_modules")
+
+genrule(
+    name = "protos",
+    outs = ["dist"],
+    cmd = """
+    mkdir -p $(OUTS);
```

Comment:
In general, these semicolons are not necessary. Remember you are just writing `bash` (and most of the time, you don't need semicolons to end statements in `bash`)